Assets/MRTK/Core/Utilities/PlatformUtility.cs

```diff
@@ -7,9 +7,13 @@ namespace Microsoft.MixedReality.Toolkit.Utilities
 {
     public static class PlatformUtility
     {
-        public static bool IsPlatformSupported(this RuntimePlatform runtimePlatform, SupportedPlatforms platforms)
```

Comment:
Whoops, not sure where my originally referenced comment went:

To prevent this from being a breaking change for any dev who might use this API in their own app, we should keep these two methods around and tag them with `[Obsolete("Use PlatformUtility.IsPlatformSupported instead, which accounts for both the in-editor and runtime case.")]